Meta/Lagom/ReadMe.md

```diff
@@ -130,3 +130,28 @@ You may run into annoying issues with the stacktrace:
 
 That means it couldn't find the executable `llvm-symbolizer`, which could be in your OS's package `llvm`.
 `llvm-symbolizer-11` will [not be recognized](https://stackoverflow.com/a/42845444/).
+
+## Using Lagom in an External Project
+It is possible to use Lagom for your own projects outside of Serenity too\!  
+An example of this in use can be found [on Linus' LibJS test262 runner](https://github.com/linusg/libjs-test262).
+
+To implement this yourself:
+- Download a copy of [linusg/libjs-test262/cmake/FetchLagom.cmake](https://github.com/linusg/libjs-test262/blob/7832c333c1504eecf1c5f9e4247aa6b34a52a3be/cmake/FetchLagom.cmake) and place it wherever you wish
+- In your root `CMakeLists.txt`, add the following commands:
+```cmake
+include(FetchContent)
+include(cmake/FetchLagom.cmake) # If you've placed the file downloaded above differently, be sure to reflect that in this command :^)
+```
+- In addition, you will need to also add some compile options that Serenity uses to ensure no warnings or errors:
+```cmake
+add_compile_options(-Wno-literal-suffix) # AK::StringView defines operator"" sv, which GCC complains does not have an underscore.
+add_compile_options(-fno-gnu-keywords)   # JS::Value has a method named typeof, which also happens to be a GNU keyword.
```

Comment:
Is that actually necessary? We don't seem to be using it in the libjs-test262 runner...

Reply:
I've just tested it and unless I'm doing something wrong, I did need to:
```cpp
ErrorOr<int> serenity_main(Main::Arguments arguments)
{
    JS::Value some_value(3);

    outln("Some value is type {}", some_value.typeof());

    return 0;
```

Does not compile without it:
- `Userland/Libraries/LibJS/Runtime/Value.h:345:19: error: expected primary-expression before ‘)’ token`
- `Userland/Libraries/LibJS/Runtime/Value.h:345:5: error: multiple types in one declaration`
- `Userland/Libraries/LibJS/Runtime/Value.h:345:21: error: declaration does not declare anything [-fpermissive]`
- Inside the test binary, `main.cpp:21:26: error: expected unqualified-id before ‘typeof’`